code/gateway/kxdash.q

```diff
@@ -24,20 +34,21 @@ dashjoin:{[joinfunc;r]
   (`.dash.snd_err;r[0;1;`w];r[0;1;`r];r[0;1;`result])]
  }
 
+//Define the current .z.ps as a new function to be called in dashps
+.z.ops:@[.:;`.z.ps;{.:}];
```

Comment:
don't use .z name space for custom functions
Also I think that should be value rather than .: